package.json

```diff
@@ -33,10 +36,16 @@
   },
   "homepage": "https://github.com/gagern/node-libtidy#readme",
   "devDependencies": {
+    "@types/chai": "^3.5.0",
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
Sorry to say this, but the `@types/mocha` package does not have a `3.x` version. The version `2.2` still it covers the mocha API we need.
Should I use this mismatched version, or remove this package and declare a minimal mocha API in place?